src/machine/arithmetic_ops.rs

```diff
@@ -1067,16 +1067,20 @@ pub(crate) fn truncate(n: Number, arena: &mut Arena) -> Number {
     }
 }
 
-pub(crate) fn round(n: Number, arena: &mut Arena) -> Result<Number, MachineStubGen> {
-    let stub_gen = || {
-        let is_atom = atom!("is");
-        functor_stub(is_atom, 2)
+pub(crate) fn round(num: Number, arena: &mut Arena) -> Result<Number, MachineStubGen> {
```

Comment:
This function is now infallible (since we're not doing `+ 0.5`), but I don't want to weigh down this PR, especially given that I'm already doing so in my refactor of `arithmetic_ops` on a separate branch